src/components/map.js

```diff
@@ -256,25 +263,23 @@ let Map = class Map extends React.Component {
         sourceLayer: "original",  
         layout: {},
         paint: {
-          /*['case',
-          ['boolean', ['feature-state', 'click'], false],
-          '#696969',
-  
-          ]*/
           "fill-color": {
             property: this.props.active.property,
             stops: this.props.active.stops,
           },
-          "fill-opacity": [
-            "case",
-            ["boolean", ["feature-state", "click"], false],
-            1,
-            ["boolean", ["feature-state", "highlight"], false],
-            1,
-            ["boolean", ["feature-state", "hover"], false],
-            1,
-            0.8,
-          ],
+          "fill-opacity": 0.8,
+        },
+      });
+      this.map.addLayer({
+        id: "sa2-borders",
+        type: "line",
+        source: "sa2-comp",
+        sourceLayer: "original",
+        layout: {},
+        paint: {
+          "line-color": "#000080",
+          "line-width": 2,
+          "line-opacity": 1.5,
```

Comment:
I get `Error: layers.mini-borders.paint.line-opacity: 1.5 is greater than the maximum value 1`. 1.0 would match the Figma mockup. I don't know why it's 1.5 elsewhere in the app and how that doesn't produce errors in the console.

Also, `line-color` should be `#ffffff`.